hooks/event-conversion-utils.ts

```diff
@@ -0,0 +1,125 @@
+import Airtable from 'airtable'
+import { CommunityEvent, CommunityEventType, WorldLocation } from '~/store/modules/events.ts'
+
+export {
+  fetchCommunityEvents,
+  convertToCommunityEvent,
+  getName,
+  getType,
+  getImage,
+  getPlace,
+  getLocation,
+  getWebsite,
+  getDates,
+  formatDates
+}
+
+async function fetchCommunityEvents (apiKey: string, { days }): Promise<CommunityEvent[]> {
+  const events: CommunityEvent[] = []
+  const base = new Airtable({ apiKey }).base('appkaaRF2QdwfusP1')
+  await base('Events Master View').select({
+    fields: [
```

Comment:
These fields are used more than once so it would be better to create constants with them. For example:
```
const NAME = 'Name'
const START_DATE='Start Date'
...
```